src/styles/components/contextmenu.less

```diff
@@ -17,12 +17,9 @@
 .context-menu-container {
   position: absolute;
   z-index: 999;
-  width: 120px;
+  width: 103px;
```

Comment:
wondering if this will be enough space if we get some longer action names...

Reply:
will make it a little bigger